Composer/packages/client/src/recoilModel/dispatchers/publisher.ts

```diff
@@ -99,6 +102,13 @@ export const publisherDispatcher = () => {
     if (data == null) return;
     const { set, snapshot } = callbackHelpers;
     const { endpointURL, status } = data;
+
+    // remove job id in publish storage if published
+    if (status === PUBLISH_SUCCESS || status === PUBLISH_FAILED) {
+      const publishJobIds = publishStorage.get('jobIds') || {};
+      publishJobIds[`${projectId}-${target.name}`] = '';
```

Comment:
maybe delete publishJobIds[`${projectId}-${target.name}`] is better?

Reply:
Yep, refactor it